### PR TITLE
Match `MSL/math`

### PR DESCRIPTION
**Pull request by FluentCoding**
Every function of math has been fully matched and decompiled except for func_803265A8. (which I'll do very soon, unless one of you wants to pick it up for themselves)

The currently linked math.o file is from the asm so this has to be changed as well as soon as func_803265A8 is matched.

- [ ] Match func_803265A8
- [ ] Rename variables to what they do
- [ ] Reformat function bodies/Remove comments
- [x] Rename functions to what they do
- [ ] Link math.o from "src"

### CODE REVIEW

**Comment by EpochFlame** [comment]
#241 accomplishes most of the labeling

File: src/MSL/math.c [hunk 1]
```diff
@@ -0,0 +1,166 @@
+#include "./math.h"
```
Comment (PsiLupan) [inline src/MSL/math.c:1]: When you have the chance, fix this include to be the explicit path: "MSL/math.h"

Comment (FluentCoding) [reply src/MSL/math.c]: done https://github.com/doldecomp/melee/pull/235/commits/ed6194cefb838f1ef1b3ca7e05e1973bcdfce662

**Comment by ribbanya** [comment]
Merge conflicts resolved in FluentCoding#2.

**Review by ribbanya — CHANGES_REQUESTED** [review]
Waiting on this to be added to `obj_files.mk`.

**Review by ribbanya — APPROVED** [review]
This is up-to-date with `master`. All data and functions contained in this PR have been integrated into C files. Some of the functions are `NON-MATCHING` but further progress can be made in a new PR. Items in the OP will be tracked in a new issue.

**Pull request merged**